BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00319-03
\ No newline at end of file
+1.0.25-prerelease-00322-04
```

Comment:
I notice CoreFx is still on 321, and it's always a little risky to be different than them.  We might want to coordinate with them or submit a PR to CoreFx to go in at the same time as we update to 322.